internal/resolution/pm/gradle/setup.go

```diff
@@ -15,8 +15,8 @@ import (
 
 const (
 	initGradle               = "gradle"
-	multiProjectFilename     = ".debricked.multiprojects.txt"
-	gradleInitScriptFileName = ".gradle-init-script.debricked.groovy"
+	multiProjectFilename     = "debricked.multiprojects.txt"
+	gradleInitScriptFileName = "gradle-init-script.debricked.groovy"
```

Comment:
Theese 2 don't need to be updated, since they're just files used in the process. 

It is instead inside the gradle-init-script.groovy on line 16 in debricked/cli/internal/resolution/pm/gradle/gradle-init/gradle-init-script.groovy, it still generates .gradle.debricked.lock

Reply:
multiproject is created during the process, but will be deleted afterwards. You need to change .debricked.multiprojects.txt in the above file as well if you want it to work.